actors/power/src/state.rs

```diff
@@ -68,6 +70,8 @@ pub struct State {
     /// Number of miners having proven the minimum consensus power.
     pub miner_above_min_power_count: i64,
 
+    pub epochs_since_ramp_start: i64,
```

Comment:
If I recall our convo correctly @lemmih - this will be hardcoded for each network at the next upgrade?

Reply:
So I think the main thing remaining is - where does the epochs_since_ramp_start parameter get updated?  It seems there needs to be some way to access the current epoch of the network as well as a hardcoded value of the epoch at which the upgrade will take place.